src/main.lua

```
@@ -969,6 +969,10 @@ local function num_in_ranges(num, ranges, size)
 end
 
 local function has_tag(tag, name)
+  if tags_cache[name] == nil then
+    -- For some reason this happens for fluids
+    return false
+  end
```

I suspect there's other cases that were missed besides fluids, and with this change they'll now fail silently. Can you make it throw an error if tags_cache[name] is nil and instead make it handle fluids specifically in chest_wrap?

---

I tried but this wasn't working properly. I will try to improve it later